iotdb-core/metrics/interface/src/main/java/org/apache/iotdb/metrics/metricsets/jvm/JvmGcMetrics.java

```diff
@@ -116,17 +163,67 @@ public void bindTo(AbstractMetricService metricService) {
             String gcAction = notificationInfo.getGcAction();
             GcInfo gcInfo = notificationInfo.getGcInfo();
             long duration = gcInfo.getDuration();
```

Comment:
When does the time get wrong here? Can you add some comments to list an example?